rts/at/sra/README.md

```diff
@@ -0,0 +1,68 @@
+OVERVIEW
+--------
+An example of how to use REST-AT service.
+
+USAGE
+--------------------
+TEST SRA PARTICIPANTS
```

Comment:
I know each of the standard section headers of the README are meant be capitalised, but why do these lines need to be capitalised?

Reply:
Those are also header so I used capitalised. Those looks different.

Reply:
They aren't headers. There are many resources on the web for learning markdown, for example https://www.markdownguide.org/basic-syntax/#headings includes a definition of the markdown header syntax.

Also the standard style that we use for the rts READMEs is

```
OVERVIEW
USAGE
EXPECTED OUTPUT
WHAT JUST HAPPENED?
```

as I mentioned in a previous review (https://github.com/jbosstm/quickstart/pull/434#discussion_r1190171503). If you do need to add extra headers (but try to avoid adding too many, preferring the use of normal prose instead) then add them as sub-headers of this style.